### DISCUSSION TITLE
touchTap regression in 6.2

### DISCUSSION THREAD

**Discussion by lhoward** [elgatosf/elgato#16] [Help]
A couple of issues with my plugin since upgrading to 6.2, specifically with the SD+.

First, you can't clear an icon with an empty SVG `<svg></svg>`, it requires a width and height. Easy to fix on my end.

More concerning is the long touches (touchTap with `hold` set to true) are not working. On a long touch, I receive a touchTap event with `hold: false`, and then on the next short touch, an event with `hold: true`. This breaks my plugin. Any ideas?

**Comment by GeekyEggo** [comment]
Hi @lhoward, the touch tap scenario you're experiencing sounds like an issue we're familiar with, whereby events occur one-step behind. Could I ask you to disconnect your Stream Deck+ from your computer, reconnect it, and test again. If this doesn't resolve the issue, please let me know.

With regards to the SVG; IIRC, the requirement of a `width` and `height` were introduced as part of a previous version (possibly version 6). May I ask which version you were upgrading from? I apologise for any inconvenience the change has caused, and to resolve the issue you should be able to assign a clear SVG using `<svg width="1" height="1" />`.

**Comment by lhoward** [comment]
Thanks! Just noticed regression with 6.2, 6.1 was fine as far as I’m aware. I will try the disconnect/reconnect and let you know.

**Comment by lhoward** [comment]
Yup, disconnect/reconnect fixes issue.
